### PR TITLE
add option to inactivate a lab/distcenter - needs more work

### PR DESCRIPTION
**Pull request by morfi11**
@noam-honig - need your help in implementation of isDistCenterInactive - how to get the distcenter fields by DistCenterId reference in the column.

### CODE REVIEW

File: src/app/families/FamilyDeliveries.ts [hunk 1]
```diff
@@ -92,6 +92,9 @@ export class FamilyDeliveries extends IdEntity {
     distributionCenter = new DistributionCenterId(this.context, {
         allowApiUpdate: Roles.admin
     });
+    isDistCenterInactive() {
+        return true; // this.distributionCenter.isNotActive;
```
Comment (morfi11) [inline src/app/families/FamilyDeliveries.ts:96]: @noam-honig : how do I get to this field from here?

File: src/app/manage/distribution-centers.ts [hunk 2]
```diff
@@ -21,9 +21,8 @@ export class DistributionCenters extends IdEntity {
   phone1Description = new StringColumn(getLang(this.context).phone1Description);
   phone2 = new PhoneColumn(getLang(this.context).phone2);
   phone2Description = new StringColumn(getLang(this.context).phone2Description);
-
-
-
+  isNotActive = new BoolColumn(getLang(this.context).frozen);
```
Comment (morfi11) [inline src/app/manage/distribution-centers.ts:24]: added as negative logic assuming the default for bool is false and will not affect existing systems.

Comment (noam-honig) [reply src/app/manage/distribution-centers.ts:24]: Why not stay consistent with the archive concept that is used for helpers and FamilyDeliveries?

File: src/app/families/FamilyDeliveries.ts (hunk 1 above)
Comment (noam-honig) [reply src/app/families/FamilyDeliveries.ts:96]: It depends on the use case. Why do you need it here?

Comment (noam-honig) [reply src/app/families/FamilyDeliveries.ts:96]: this.context.for(DistributioCenter).lookup(this.DistributionCenter).IsNotActive.Value

**Pull request merged**